src/deb/control/postrm

```diff
@@ -1,5 +1,55 @@
-#!/bin/bash
-echo 'postrm script executing'
+#!/usr/bin/env bash
 
-# delete any leftover flyway artifacts
-rm -rf /usr/share/nedinstall/bin/flyway-*
+set -e
+
+NAME=ned
+
+TOMCAT7_USER=$NAME
+TOMCAT7_GROUP=$NAME
+HOME=/opt/plos/$NAME
+CONFFILE=/etc/default/$NAME
+
+if [ "$1" = purge ] && [ -e /usr/share/debconf/confmodule ]; then
+	. /usr/share/debconf/confmodule
+	db_purge
+fi
+
+if [ "$1" = "purge" ] ; then
+	update-rc.d $NAME remove >/dev/null
+fi
```

Comment:
Is there a reason the above `purge` conditionals are not part of the case statement below?